### DISCUSSION TITLE
Release the SD card

### DISCUSSION THREAD

**Discussion by yovanf** [schreibfaul1/ESP32-audioI2S#419] [Q&A]
I am using an ESP32
Once an mp3 file has been acccess using  audio.connecttoFS(SD,char_filename); If you try to access the SD card e.g to list all files present on the SD CARD the ESP crashes(see below)  . Is there a way for the library to release the SD card ??
  
Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.
17:35:18.140 -> Core 1 register dump:
17:35:18.140 -> PC      : 0x4000c3f0  PS      : 0x00060030  A0      : 0x800ff381  A1      : 0x3ffe8bb0  
17:35:18.232 -> A2      : 0x3fff9ad0  A3      : 0x00000000  A4      : 0x00000001  A5      : 0x0000ff00  
17:35:18.324 -> A6      : 0x00ff0000  A7      : 0xff000000  A8      : 0x00000000  A9      : 0x00000000  
17:35:18.416 -> A10     : 0x3ffe8c44  A11     : 0x3fffb0d5  A12     : 0x0000386a  A13     : 0x0000ff00  
17:35:18.508 -> A14     : 0x00ff0000  A15     : 0xff000000  SAR     : 0x00000018  EXCCAUSE: 0x0000001c  
17:35:18.600 -> EXCVADDR: 0x00000000  LBEG    : 0x400014fd  LEND    : 0x4000150d  LCOUNT  : 0xfffff1e5  
17:35:18.691 -> 
17:35:18.691 -> Backtrace: 0x4000c3f0:0x3ffe8bb0 0x400ff37e:0x3ffe8bd0 0x400ff49a:0x3ffe8bf0 0x400facdd:0x3ffe8c10 0x400f95f2:0x3ffe8c80 0x400f9555:0x3ffe8cc0 0x400dcd26:0x3ffe8ce0 0x400df0a9:0x3ffe9300 0x400fc481:0x3ffe9320 0x400fa069:0x3ffe9360 0x400fa12d:0x3ffe93a0 0x400fa395:0x3ffe93e0 0x400f7499:0x3ffe9400 0x400f7519:0x3ffe9440 0x400f7b62:0x3ffe9460 0x400896f5:0x3ffe9490
17:35:19.105 -> 
17:35:19.105 -> Rebooting...

**Comment by schreibfaul1** [comment]
With SD is nothing. Please use SD_MMC to open multiple instances at the same time.

**Comment by yovanf** [comment]
Hi , Using ( SD_MMC,char_filename) , there is no audio that is outputted
